wireless_modules/wireless_module.py

```diff
@@ -152,7 +179,7 @@ async def start_data_loop(self, interval):
         stop message - after which the process is repeated.
         :param interval: Integer representing number of seconds to wait before sending data.
         """
-        secs_to_ms = 1000
+        secs_to_ms = 200
```

Comment:
Should `secs_to_ms` be renamed to something like `time_scalar`... as 200ms isn't really 1 second (being a bit pedantic :) )?